package/batocera/core/batocera-system/Config.in

```diff
@@ -1418,7 +1418,7 @@ config BR2_PACKAGE_BATOCERA_GUNS
 	select BR2_PACKAGE_SAMCO_GUNS				# samco guns
 	select BR2_PACKAGE_FUSION_LIGHTGUNS			# fusion lightguns
 	select BR2_PACKAGE_DOLPHINBAR_GUNS			# dolphinbar guns
-	#nope select BR2_PACKAGE_SINDEN_GUNS			# sinden guns
+	#nope select BR2_PACKAGE_SINDEN_GUNS			# sinden guns (Needs to pull and build mono from scratch !!!!)
```

Comment:
For weaker boards and portable consoles that do not have an HDMI output (example rk3326) we can disable external weapons and controls.